data/mods/My_Sweet_Cataclysm/sweet_monsters.json

```
@@ -387,7 +387,7 @@
     "id": "mon_gum_spider_wolf",
     "type": "MONSTER",
     "name": { "str": "caffeinated gum spider" },
-    "description": "A giant piece of gum streched in the shape of a spider.  It moves quickly and aggressively as if under the effect of some stimulant.",
+    "description": "A giant piece of gum stretched in the shape of a spider.  It jitters and darts about rapidly as if affected by its own caffeine content.",
```

```suggestion
    "description": "A giant piece of gum stretched in the shape of a spider.  It jitters and darts about rapidly as if it had too much energy to stand still.",
```
Let's try to be a little less on the nose with the caffeine stuff